src/mudlet.h

```diff
@@ -70,7 +70,7 @@ class mudlet : public QMainWindow, public Ui::MainWindow
    static                        mudlet * self();
    // This method allows better debugging when mudlet::self() is called inappropriately.
    static                        void start();
-   HostManager *                 getHostManager();
+   HostManager& getHostManager() { return mHostManager; }
```

Comment:
In this case, part of the point is that the caller modifies it, though, I guess only once, to add a host.